src/js/Controllers/DisplayCapabilities.js

```diff
@@ -540,6 +550,7 @@ let capabilities = {
 				imageField("choiceSecondaryImage", 80),
 				imageField("cmdIcon", 85),
 				imageField("subMenuIcon", 85),
+				imageField("secondaryImage", 40),
```

Comment:
This ImageFieldName does not exist, and it looks like `choiceSecondaryImage`, `menuCommandSecondaryImage`, and `menuSubMenuSecondaryImage` are already here

Reply:
Ah, I suppose this means it should be removed from all of the other templates as well, didn't realize it was copied over.